### ISSUE TITLE
Road rehabilitation in South Sudan by UNMISS on 2020 - 2021

### ISSUE THREAD

**Issue by ElenaGallo** [geosolutions-it/UNMISS-GeoStory#1]
* Data Review [here](https://docs.google.com/spreadsheets/d/1nR9B-Hx6cjJQ7rOxN46xRDs_VAVjwuI38FyWp-LYDRI/edit?usp=sharing)
* Work in progress geostory [here](https://stable.demo.geonode.org/apps/2837/edit#/)

**Comment by simboss** [comment]
@ElenaGallo we discussed with UNMISS about the story you created.

First of all, we got more information about the data, which is attached [here](https://github.com/geosolutions-it/UNMISS-GeoStory/files/6961964/UNMISS.Road.Rehabilitation.Data.Explanation.pdf).
Notice that we need to update the maps to reflect the styling they are presenting; not sure we will be able to do the double stroking so we can do the single stroking.

They will also came up with more feedback which I will provide later on, so that will be captured in a successive iteration.
